tests/auth_tests/test_forms.py

```diff
@@ -778,6 +778,13 @@ def test_username_field_autocapitalize_none(self):
         form = UserChangeForm()
         self.assertEqual(form.fields['username'].widget.attrs.get('autocapitalize'), 'none')
 
+    def test_password_field_disabled(self):
+        form = UserChangeForm()
+        # Ensures that the password field is disabled and is
+        # considered unchanged regardless of the specified values
```

Comment:
IMO we don't need this comment. Also, to add more explanation, comments should state the expected behavior and omit prefixes like "Ensure that" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).